packages/core/src/internal/utils/dom.spec.ts

```diff
@@ -833,4 +834,15 @@ describe('Functional Helper: ', () => {
       expect(test.textContent).toBe('ohai', 'fragment has our text');
     });
   });
+
+  describe('coerceBooleanProperty():', () => {
+    it('should coerse a value to a boolean', () => {
+      expect(coerceBooleanProperty(null)).toBe(false);
+      expect(coerceBooleanProperty(undefined)).toBe(false);
+      expect(coerceBooleanProperty(false)).toBe(false);
+      expect(coerceBooleanProperty(true)).toBe(true);
+      expect(coerceBooleanProperty('false')).toBe(false);
+      expect(coerceBooleanProperty('true')).toBe(true);
```

Comment:
probably need a test for empty string just to have it

Reply:
ah saw this after the merge, github was stale I guess. I'll add one to be safe with a follow up PR